biztalk/install-and-config-guides/whats-new-in-biztalk-server-2020.md

```diff
@@ -35,10 +35,10 @@ Read about what's new in BizTalk Server 2020.
 | Office 365 Adapters | Leverage the power of BizTalk in newer office automation workflows by integrating with Adapters that allows you to send or receive Office 365 emails, receive or transmit Office 365 calendar events and create Office 365 contacts.<br/><br/>[Office 365 Outlook Email adapter](../core/office365-mail-adapter.md)<br/>[Office 365 Outlook Calendar adapter](../core/office365-calendar-adapter.md)<br/>[Office 365 Outlook Contact adapter](../core/office365-contact-adapter.md)|
 |Group Managed Service Accounts|Extend windows GMSA support to BizTalk operations and services. <br/><br/>[Using Group Managed Service Account](../core/gmsa-config.md)|  
 |Blob Adapter|Send and Receive messages to/from Azure Blob Storage. <br/><br/>[Azure Blob storage adapter](../core/azure-blob-adapter.md)|  
-|Audit Log|Making BizTalk further secure by maintaining audit trails of all management operation.|  
+|Audit Log|Making BizTalk further secure by maintaining audit trails of all management operation.<br/><br/>[Configuring and Viewing Audit Logs](https://docs.microsoft.com/en-us/biztalk/core/audit-management-operations)||  
 |New Read Only Operator role|Brand new read only operator role to facilitate dev ops model, where access to production stamp is provided without the ability to update anything.|  
 |XSLT 3.0|New extensible model for runtime map execution, out of box wiring to work with Saxon XSLT3.0. <br/><br/>[XSLT Transform Engine](https://docs.microsoft.com/biztalk/core/technical-reference/xslt-transform-engine-grid-property)<br/>[XSLT custom transform implementation](https://docs.microsoft.com/biztalk/core/technical-reference/xslt-custom-transform-implementation)|  
-|Additional updates|<ul><li>Move to new long term supported Microsoft OLEDB Driver for SQL Server. </li><br/><li>Support for SSO Affiliate applications in SFTP adapter.</li><br/><li>SQL Availability Group support for BAM DTS Package via SSIS Catalog.</li><br/><li>Partially disabled receive locations.</li><br/><li>Throughput improvements for Dynamic Send Ports with Ordered Delivery.</li></ul>|   
+|Additional updates|<ul><li>Move to new long term supported Microsoft OLEDB Driver for SQL Server. </li><br/><li>Support for SSO Affiliate applications in SFTP adapter.</li><br/><li>SQL Availability Group support for BAM DTS Package via SSIS Catalog.</li><br/><li>Partially disabled receive locations.<br/>[How to Enable Receive Location Fault Tolerance](https://docs.microsoft.com/en-us/biztalk/technical-guides/how-to-enable-receivelocation-faulttolerance)</li><br/><li>Throughput improvements for Dynamic Send Ports with Ordered Delivery.</li></ul>|   
```

Comment:
use relative path to link to file:
(../technical-guides/how-to-enable-receivelocation-faulttolerance.md)